lib/paperclip_processors/cropper.rb

```diff
@@ -0,0 +1,25 @@
+# Reference: http://railscasts.com/episodes/182-cropping-images
```

Comment:
⚠️ Wasn't sure of the best way to test this file... 🤔

Reply:
curious: does this get hit with your ui tests?

Reply:
I don't believe it does... Since the image remains uncropped when hitting save.

Reply:
I don't believe so... since I don't see the expected cropped photo result after saving